<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ana Sayfa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WhatsApp-Image-2022-05-18-at-12.38.31.ico</normaloff>WhatsApp-Image-2022-05-18-at-12.38.31.ico</iconset>
  </property>
  <widget class="QPushButton" name="BtnIptal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt; Ana Menü</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Girilecek Değerler</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>167</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>T01 (Çıkış anındaki sıcaklık değeri) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>165</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P01 (Çıkış anındaki basınç değeri) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                         M1 (Mach Sayısı) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>145</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>            m1 (Kütlesel debi ağırlığı) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>183</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>                                              alfa : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rm (Aft çapının uzunluğu) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>258</y>
      <width>159</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                         gama : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtT01">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>41</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtP01">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>77</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtM1">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>113</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtm1">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>149</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlfa">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>185</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRm">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>221</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGama">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>257</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Stage 1 Sonuçlar</string>
   </property>
   <widget class="QLineEdit" name="txtTs1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>31</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPs1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>57</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtg1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>83</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txta1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>109</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtV1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>135</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtH1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>213</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRt">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>239</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRh">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>265</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtArea1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>187</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtVx1">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>161</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>34</y>
      <width>168</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ts1 (Giriş anındaki sıcaklık değeri) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>60</y>
      <width>166</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ps1 (Giriş anındaki basınç değeri) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                g1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>111</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                a1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>213</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                               H1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>187</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                          Area1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>136</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                V1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>162</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                              Vx1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>264</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                               Rh : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>238</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                Rt : </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnTemizle">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Temizle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnSonraki">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Sonraki Sayfa &gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtT01</tabstop>
  <tabstop>txtP01</tabstop>
  <tabstop>txtM1</tabstop>
  <tabstop>txtm1</tabstop>
  <tabstop>txtAlfa</tabstop>
  <tabstop>txtRm</tabstop>
  <tabstop>txtGama</tabstop>
  <tabstop>BtnIptal</tabstop>
  <tabstop>txtg1</tabstop>
  <tabstop>txtPs1</tabstop>
  <tabstop>txtTs1</tabstop>
  <tabstop>txtV1</tabstop>
  <tabstop>txtH1</tabstop>
  <tabstop>txtRt</tabstop>
  <tabstop>txtRh</tabstop>
  <tabstop>txtArea1</tabstop>
  <tabstop>txtVx1</tabstop>
  <tabstop>txta1</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
